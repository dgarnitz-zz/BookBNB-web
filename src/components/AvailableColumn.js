import React from 'react';
import TableCell from '@material-ui/core/TableCell';

function AvailableColumn(props){

  return (
    <TableCell>
      Available
    </TableCell>
  );
}

export default AvailableColumn;
