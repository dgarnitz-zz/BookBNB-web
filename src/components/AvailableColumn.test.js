import React from 'react';
import AvailableColumn from './AvailableColumn';
import renderer from 'react-test-renderer';

test('test_one', () => {
  const component = renderer.create(
    <AvailableColumn />
  )

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
