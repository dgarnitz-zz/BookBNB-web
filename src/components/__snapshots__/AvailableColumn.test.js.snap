// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test_one 1`] = `
<td
  aria-sort={null}
  className="MuiTableCell-root-1"
>
  Available
</td>
`;
